tun/tests/configure.rs

```diff
@@ -19,4 +19,16 @@ fn test_set_get_ipv4() {
     let result = tun.ipv4_addr()?;
 
     assert_eq!(addr, result);
-}
\ No newline at end of file
+}
+
+#[test]
+#[throws]
+fn test_set_get_ipv6() {
+    let tun = TunInterface::new()?;
+
+    let addr = Ipv6Addr::new(1, 1, 1, 1, 1, 1, 1, 1);
+    tun.set_ipv6_addr(addr)?;
+
+    // let result = tun.ipv6_addr()?;
+    // assert_eq!(addr, result);
+}
```

Comment:
Why is this commented out?

Reply:
These lines are commented out because we found out that "tun.ipv6_addr()" didn't work - for some reason setting the IPV6 address is supported but not getting it. cc: @conradev could you share the documentation where we read that?